app/src/pages/inside/settingsPage/settingTabs/notificationsTab/forms/emailCasesList.jsx

```diff
@@ -0,0 +1,170 @@
+        },
+      });
+    } else {
+      this.removeEmailCase(id);
+    }
+  };
+  onSubmit = (data) => {
+    const emailCase = this.validateEmailCase(data);
+    const { valid } = emailCase;
+    const emailCases = this.editEmailCasesList({
+      ...emailCase,
+      submitted: valid,
+      showValidationMessage: !valid,
+    });
+    this.setState({ emailCases }, this.submit);
+  };
+  onChange = (data) => {
+    const emailCase = this.validateEmailCase(data);
+    const emailCases = this.editEmailCasesList(emailCase);
+    this.setState({ emailCases });
+  };
+  submit = () => {
+    this.props.onSubmit({
+      emailCases: this.state.emailCases.map(this.convertEmailCasesForSubmission),
+    });
+  };
+  addEmailCase = () => {
+    const emailCases = [
+      ...this.state.emailCases,
+      { ...this.validateEmailCase(defaultRecipient), id: uniqueId() },
+    ];
+    this.setState({ emailCases });
+  };
+  removeEmailCase = (id) => {
+    const emailCases = this.state.emailCases.filter((item) => item.id !== id);
+    this.setState({ emailCases }, this.submit);
+  };
+  editEmailCasesList = (emailCase) =>
+    this.state.emailCases.map((item) => (item.id === emailCase.id ? emailCase : item));
+  validateEmailCase = (data) => {
+    const { emailCases } = this.state;
+    const { intl } = this.props;
+    const hasDuplicates = emailCases
+      .filter((item) => item.id !== data.id)
+      .some((item) =>
+        compareObjectsByKeys(item, data, [
+          'informOwner',
+          'launchNames',
+          'recipients',
+          'sendCase',
+          'tags',
+        ]),
+      );
+    return {
+      ...data,
+      valid: !hasDuplicates,
+      validationMessage: intl.formatMessage(messages.duplicationErrorMessage),
+    };
+  };
+  convertEmailCasesForSubmission = (obj) => {
+    const { informOwner, launchNames, recipients, sendCase, tags } = obj;
+    return {
+      launchNames,
+      sendCase,
+      tags,
+      recipients: informOwner ? [...recipients, OWNER] : recipients,
+    };
+  };
+  render() {
+    const { intl, configurable } = this.props;
+    const { emailCases } = this.state;
+    const emailCasesLength = emailCases.length;
+
+    return (
+      <Fragment>
+        {emailCases.map((item, id) => {
+          const emailCasesProps = {
+            form: `notificationRulesForm_${item.id}`,
+            key: item.id,
+            initialValues: item,
+            emailCase: item,
+            index: id + 1,
+            onDelete: this.onDelete,
+            onSubmit: this.onSubmit,
+            onChange: this.onChange,
+            configurable: this.props.configurable,
+            deletable: emailCasesLength > 1,
+          };
+          return <EmailCase {...emailCasesProps} />;
+        })}
+        {!configurable && (
+          <div className={cx('notification-form-button')}>
+            <GhostButton onClick={this.addEmailCase}>
+              + {intl.formatMessage(messages.addNewRuleButton)}
```

Comment:
If you need to add plus symbol, please provide svg with icon prop of GhostButton.

Reply:
Please ask our UX Designer Iryna Melnyk to get that icon in SVG

Reply:
Done